src/overlay/test/OverlayManagerTests.cpp

```diff
@@ -285,13 +285,6 @@ class OverlayManagerTests
         crank(10);
         std::vector<int> expectedFinal{2, 2, 1, 2, 2};
         REQUIRE(sentCounts(pm) == expectedFinal);
-
```

Comment:
Thanks! I did some archeology and this was added for CAP-0015 transition and we forgot to remove this as part of the post rollout cleanup